src/05-objects.js

```diff
@@ -22,7 +24,13 @@ function createEmptyObject() {}
  *  createObjectWithValue(19);
  *  //> { whateverKey: 19 }
  */
-function createObjectWithValue(val) {}
+function createObjectWithValue(val) {
+  let blankObject = {}; 
```

Comment:
Nice work here - we could also put this in one line if we wanted:  ` return { whateverKey: val } `.